models/wiki.go

```diff
@@ -42,6 +42,12 @@ func WikiNameToFilename(name string) string {
 	return url.QueryEscape(name) + ".md"
 }
 
+// WikiNameToFilenameRaw converts a wiki name to its corresponding raw filename.
+func WikiNameToFilenameRaw(name string) string {
+	name = strings.Replace(name, " ", "-", -1)
+	return url.QueryEscape(name)
+}
+
 // WikiFilenameToName converts a wiki filename to its corresponding page name.
 func WikiFilenameToName(filename string) (string, error) {
```

Comment:
`WikiNameToFilename` should reuse `WikiNameToFilenameRaw`

Reply:
Thanks for the review.
Now that I look at the code of models/wiki.go again I can't see the difference between `WikiNameToFilenameRaw`and `WikiNameToSubURL`. I think I totally duplicated a method that was already in place... Doh.
I will fix that.